v1/summary.yaml

```diff
@@ -145,15 +145,16 @@ paths:
             request:
               method: put
               uri: /{domain}/sys/key_value/page_summary/{request.params.title}
-              headers: '{{merge({"if-none-hash-match": "*"}, extract.headers)}}'
+              headers: '{{merge({"if-none-hash-match": "*"}, extract.headers, "cache-control": request.headers.cache-control)}}'
```

Comment:
Why is the request's `cache-control` being added here and below?

Reply:
@d00rman to pass through the `no-store` header.